components/ws-manager/pkg/manager/status.go

```diff
@@ -486,6 +502,13 @@ func (m *Manager) extractStatusFromPod(result *api.WorkspaceStatus, wso workspac
 			return nil
 		}
 
+		// the pod phase of pending is somtimes slipped because it may not be PENDING at the stage of retrieving information on the workspace pod.
```

Comment:
```suggestion
		// the pod phase of pending is sometimes slipped because it may not be PENDING at the stage of retrieving information on the workspace pod.
```

Reply:
Thanks! Fixed